src/components/Groups.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export default function Groups({ groupsChosen }) {
```

Comment:
I would be less specific here to allow usage in different cases

```suggestion
export default function Groups({ groups }) {
```